dockerfiles/che/entrypoint.sh

```diff
@@ -265,9 +261,7 @@ init() {
   [ -z "$CHE_WORKSPACE_STORAGE_CREATE_FOLDERS" ] && export CHE_WORKSPACE_STORAGE_CREATE_FOLDERS=false
 
   # Move files from /lib to /lib-copy.  This puts files onto the host.
-  rm -rf ${CHE_DATA}/lib/*
-  mkdir -p ${CHE_DATA}/lib  
-  cp -rf ${CHE_HOME}/lib/* "${CHE_DATA}"/lib
+  rm -rf ${CHE_DATA}/lib
```

Comment:
Please, create an issue to remove this line after several releases. And update comment since it is out of date now.